config/puma.rb

```diff
@@ -29,17 +30,35 @@
 # Allow puma to be restarted by `rails restart` command.
 plugin :tmp_restart
 
-# SSL in development (as DFE Signin redirects to https://localhost:3000 but
-# just listen on port 3000 elsewhere
 listen_port = ENV.fetch("PORT") { 3000 }
 
-if Rails.env.development?
+if env == "development"
+  cert = "#{Dir.pwd}/#{File.join('config', 'localhost', 'https', 'localhost.crt')}"
+  key = "#{Dir.pwd}/#{File.join('config', 'localhost', 'https', 'localhost.key')}"
 
-  cert = Rails.root.join('config', 'localhost', 'https', 'localhost.crt')
-  key = Rails.root.join('config', 'localhost', 'https', 'localhost.key')
+  unless File.exist?(key)
```

Comment:
Do we need to check for  `File.exist?(cert) && File.exist?(key)` ?

Reply:
Yeah may as well